tests/net/socket/misc/src/main.c

```diff
@@ -610,4 +864,11 @@ ZTEST_USER(socket_misc_test_suite, test_ipv6)
 	test_ipv6_getpeername();
 }
 
+ZTEST_USER(socket_misc_test_suite, test_ipv4_mapped_to_ipv6)
+{
+	test_ipv4_mapped_to_ipv6_disabled();
+	test_ipv4_mapped_to_ipv6_enabled();
+	test_ipv4_mapped_to_ipv6_server();
```

Comment:
Do those have to execute sequentially, i. e. couldn't they be separate test cases?

Reply:
They could be separate as they do not depend on each other. I can change this if needed.